docs/_config.yml

```diff
@@ -0,0 +1,48 @@
+# Template generated by https://github.com/coreos/repo-templates; do not edit downstream
+
+# To test documentation changes locally or using GitHub Pages, see:
+# https://github.com/coreos/fedora-coreos-tracker/blob/main/docs/testing-project-documentation-changes.md
+
+title: coreos/{{ git_repo }}
+description: {{ git_repo }} documentation
```

Comment:
This is user-facing documentation, so let's call the project by its proper name rather than by the name of its Git repo.  That'll require a new substitution variable, which maybe should go in the top-level `config.yaml`.